contrib/platform/test/com/sun/jna/platform/unix/X11Test.java

```diff
@@ -76,6 +77,24 @@ public void testXFetchName() {
         }
     }
 
+    public void testXSetWMProtocols() {
+        X11.Atom[] protocols = new X11.Atom[]{ X11.INSTANCE.XInternAtom(display, "WM_DELETE_WINDOW", false) };
+        int status = X11.INSTANCE.XSetWMProtocols(display, root, protocols, 1);
```

Comment:
Please replace the hardcoded count value of "1" with ```protocols.length```,  this enforces correct values, even if the input is changed.